dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/runner/task/subworkflow/SubWorkflowLogicTask.java

```diff
@@ -133,6 +147,43 @@ public void kill() {
         }
     }
 
+    @Override
+    public List<Property> getVarPool() {
+        List<Property> taskInstanceProps = getTaskParameters().getVarPool();
+        ProcessInstanceMap processInstanceMap = processInstanceMapDao.queryWorkProcessMapByParent(
+                taskExecutionContext.getProcessInstanceId(), taskExecutionContext.getTaskInstanceId());
+        ProcessInstance childProcessInstance = processInstanceDao.queryById(processInstanceMap.getProcessInstanceId());
+        if (!StringUtils.isBlank(childProcessInstance.getVarPool())) {
```

Comment:
```suggestion
        if (StringUtils.isBlank(childProcessInstance.getVarPool())) {
            return taskInstanceProps
        }
```

Please directly return if there is not varpool in sub workflow?

Reply:
<img width="1126" alt="image" src="https://github.com/apache/dolphinscheduler/assets/33984497/e61c87af-e168-4959-850c-942a5d0586fe">

hi @ruanwenjun  If it is empty, it will be returned in the last line of the method。 
Maybe it would be more readable if you wrote it the way you did?

Reply:
@ruanwenjun PTAL